i18n/en_AU.json

```diff
@@ -3937,7 +3937,7 @@
   "remove_next_steps_modal.confirm": "Remove",
   "remove_next_steps_modal.header": "Remove {title}?",
   "remove_next_steps_modal.helpText": "Access {title} any time through the Main Menu",
-  "remove_next_steps_modal.mainText": "This will remove this section from your sidebar, but you can access it later in the Tips and Next steps section of the Main Menu.",
+  "remove_next_steps_modal.mainText": "This will remove this section from your sidebar, but you can access it later in the {title} section of the Main Menu.",
```

Comment:
That is correct, we should not update other translations files than `en.json`. The updates in the main file will be reflected in our translation servers and the specific languages will show up to be changed as well by translators.